webpack/scenes/ContentViews/Table/ContentViewsTable.js

```diff
@@ -166,28 +166,31 @@ const ContentViewTable = () => {
       canSelectAll={false}
       cells={columns}
       autocompleteEndpoint="/content_views/auto_complete_search"
-    >
-      <>
-        <Button onClick={openForm} variant="primary" aria-label="create_content_view">
-          Create content view
-        </Button>
-        <CreateContentViewModal show={isModalOpen} setIsOpen={setIsModalOpen} aria-label="create_content_view_modal" />
-      </>
-      <>
-        <CopyContentViewModal cvId={actionableCvId} cvName={actionableCvName} show={copy} setIsOpen={setCopy} aria-label="copy_content_view_modal" />
-      </>
-      {isPublishModalOpen &&
+      actionButtons={
```

Comment:
I think adding an `actionButtons` prop is fine in the short term, but it still sorta goes against the principle of having all your JSX be clean and separated into the return or render method and not scattered among different props.  This is not a comment about this PR, but in the future I'm thinking it would be better if TableWrapper could have an API like
```js
<TableWrapper {...props} >
  <TableWrapper.TopBar>
     <TableWrapper.ActionButtons>
          <Button {...buttonProps} /> 
          <Button {...buttonProps} /> 
     </TableWrapper.ActionButtons>
  </TableWrapper.TopBar>
  <thead> ... </thead>
  <tbody> ... </tbody>
</TableWrapper>
```
I did something similar a couple years ago with the ForemanModal component. Even though that component turned out to be overkill and we should have just used Patternfly modal components all along, I think it still showed that that sort of component design works well.
@sjha4 @Andrewgdewar thoughts?